accepted/build/references-into-frameworkreferences.md

```diff
@@ -0,0 +1,59 @@
+# Supporting `Reference` items to DLLs in `FrameworkReference` targeting packs
+
+In .NET Core 3.0, we have added `FrameworkReference` items which refer to .NET Core shared frameworks (base .NET Core, ASP.NET Core, or WindowsDesktop).  At build time, the FrameworkReferences resolve to targeting packs which include the reference assemblies for the corresponding shared framework.  FrameworkReferences also make the corresponding shared framework available at runtime, either by writing the shared framework information to the runtimeconfig.json file for Framework-dependent apps, or by including the runtime in the output for self-contained apps.
+
+Here, we propose to:
+
+- Allow `Reference` items to resolve to reference assemblies inside targeting packs corresponding to the `FrameworkReference` items for a project
+- Allow specifying that the references to assemblies from a FrameworkReference's targeting pack should not be referenced by default.
+
+## Scenarios
+
+### WPF Theme Assemblies
+
+WPF has several different theme assemblies, such as PresentationFramework.Aero and PresentationFramework.Luna.  Each of these theme assemblies has types with the same name and namespace.  This means that if you reference more than one of these assemblies, you need to do so with an assembly alias in order to distinguish between the types from the different assemblies in code.
+
+With this proposal, we would not reference any of the theme assemblies by default.  To use a single theme assembly, you would add a reference to it:
```

Comment:
Not referencing any of the theme assemblies is already the default.

Reply:
We have not yet removed the theme assemblies from being referenced by default.